RealEstate/urls.py

```diff
@@ -27,6 +27,9 @@
     url(r'^login/$', CoreViews.login, name='auth_login'),
     url(r'^logout/$',
         'django.contrib.auth.views.logout_then_login', name='auth_logout'),
+    url(r'^password_change/$', 'django.contrib.auth.views.password_change',
```

Comment:
Snake case is obviously very pythonic, but my gut says hyphen-delimited is more common and easier. I'd rather have this than camel case.

Thoughts?